### PR TITLE
[WIP] Attempting to unblock tests

### PR DESCRIPTION
**Pull request by rbharath**
This PR attempts to uncomment the two tests commented out in #2264. I'm able to run both tests successfully in my local development environment without any modifications. I'm putting up a PR to see how travis responds. If you're able to repro test failures locally, please let me know! This one is hard to debug since I'm not seeing the failure locally

### CODE REVIEW

**Comment by rbharath** [comment]
The weird thing is a whole bunch of unrelated tests like `test_robust_multitask_regressor_reload()` fail now too! It's not at all clear to my why uncommenting these tests breaks unrelated tests. Is it possible the normalizing flow somehow imports something different that breaks tests?

**Comment by nd-02110114** [comment]
> The weird thing is a whole bunch of unrelated tests like test_robust_multitask_regressor_reload() fail now too! It's not at all clear to my why uncommenting these tests breaks unrelated tests. Is it possible the normalizing flow somehow imports something different that breaks tests?

`tf.keras.backend.set_floatx('float64')`  in `test_normalizing_flow_model_reload ` cause the unrelated tests' errors. 

Basically, all keras layers return the same numeric type as the input, like if the input is float32, the return is also float32. But, when adding this line,  all keras layers always return the numeric type you set like float64 in spite of the input numeric type. Therefore, the loss function raise some errors because the type is inconsistent between the input and the model's output.

I seem that @ncfrey  added `tf.keras.backend.set_floatx('float64')` for some reasons, 
but `tf.keras.backend.set_floatx('float64')` have affected globally, basically we should not use this line in our tests.

**Comment by ncfrey** [comment]
> I seem that @ncfrey added `tf.keras.backend.set_floatx('float64')` for some reasons,
> but `tf.keras.backend.set_floatx('float64')` have affected globally. Basically we should not use this line in our tests.

Really good catch @nd-02110114. I added that line to fix type errors that pop up in TensorFlow Probability. This should be fixed by casting to the correct types within the NormalizingFlow class definitions, rather than setting the backend float type. I'll put up a PR to fix this.

**Comment by rbharath** [comment]
Going to close this PR since I think we've fixed these issues in other PRs
